R/convertIRF.R

```diff
@@ -0,0 +1,29 @@
+#' convertIRF
+#'
+#' Add missing periods to data frame and interpolate missing values.
+#' The ISO codes of "IRF" data are compared with the official ISO code country list.
+#' NA values are replaced with zeros and historical data are corrected.
```

Comment:
historical data are not corrected - there's no call of `toolISOhistorical` here

Reply:
ok